tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Psi4.cpp

```diff
@@ -96,9 +96,6 @@ void test_psi_4(const RealDataType& used_for_size_real,
   auto inertial_coords =
       make_with_random_values<tnsr::I<RealDataType, 3, Frame::Inertial>>(
           nn_generator, nn_distribution, used_for_size_real);
-  for (size_t i = 0; i < 3; ++i) {
-    inertial_coords.get(i)[0] = 0.0;
-  }
```

Comment:
I get that this test as it was written doesn't work anymore...but might it be possible for you to contrive an example that does give a zero denominator after the Gram-Schmidt orthonormalization? No big deal to me if not, but I would have thought you could work out what e.g. `y_hat` is for the 0th point before normalization, then modify it so that its magnitude is in fact zero?

Reply:
If this isn't something quick to do, then I wouldn't sweat it